tests/LINEBot/Util/DummyHttpClient.php

```diff
@@ -55,8 +55,16 @@ public function get($url, array $data = [], array $headers = [])
      */
     public function post($url, array $data, array $headers = null)
     {
+        $acceptedStatusCodeEndpoints = [
+            'https://api.line.me/v2/bot/richmenu/bulk/link',
+            'https://api.line.me/v2/bot/richmenu/bulk/unlink'
+        ];
+        $statusCode = 200;
+        if (in_array($url, $acceptedStatusCodeEndpoints)) {
+            $statusCode = 202;
+        }
         $ret = call_user_func($this->mock, $this->testRunner, 'POST', $url, $data, $headers);
-        return new Response(200, json_encode($ret));
```

Comment:
DummyHttpClient return only 200 status code.
but, `/v2/bot/richmenu/bulk/link` return 202 when request succeeded.

Reply:
This method should take response status parameter optionally not deep inspecting the endpoints...

Reply:
fixed~